client/js/components/procedure/admin/DpNewProcedure/DpNewProcedure.vue

```diff
@@ -7,53 +7,288 @@
+      name="r_master"
+      value="false">
+    <!-- allow publishing of Statements by default -->
+    <input
+      type="hidden"
+      name="r_publicParticipationPublicationEnabled"
+      value="1">
+    <div v-if="hasPermission('feature_use_plis')">
+      <input
+        type="hidden"
+        name="r_name"
+        value="">
+      <input
+        type="hidden"
+        name="r_externalDesc"
+        value="">
+      <input
+        type="hidden"
+        name="r_mapExtent"
+        value="">
+    </div>
+    <fieldset>
+      <addon-wrapper hook-name="bimschg.request" />
+      <div v-if="hasPermission('feature_use_plis')">
+        <dp-form-row>
+          <dp-select
+            id="r_plisId"
+            :label="{ text: Translator.trans('name'), hint: Translator.trans('explanation.plis.procedurename') }"
+            name="r_plisId"
+            :options="plisNameOptions" />
+        </dp-form-row>
+        <dl class="u-nojs-hide--block">
+          <dt
+            v-text="Translator.trans('public.participation.desc')"
+            class="weight--bold" />
+          <dd
+            id="js__plisPlanungsanlass"
+            class="list-style-none" />
+        </dl>
+      </div>
+      <dp-form-row v-else>
+        <dp-input
+          data-cy="newProcedureTitle"
+          id="r_name"
+          :label="{ text: Translator.trans('name') }"
+          :maxlength="200"
+          name="r_name"
+          required
+          type="text" />
+      </dp-form-row>
+      <!--      {% block master %}-->
+      <dp-form-row v-if="hasPermission('feature_procedure_templates')">
+        <dp-select
+          id="blueprint"
+          :label="{
+            hint: blueprintHint,
+            text: Translator.trans('master')
+          }"
+          name="r_copymaster"
+          :options="blueprintOptions"
+          :selected="masterBlueprintId"
+          @select="setBlueprintData" />
+      </dp-form-row>
+
+<!--      {# Only show select if there is more than one choice. Otherwise, pass the id as the value of a hidden field. #}-->
+      <div v-if="procedureTypes.length > 1">
+        <dp-label
+          for="r_procedure_type"
+          :hint="Translator.trans('text.procedures.types.hint')"
+          :text="Translator.trans('text.procedures.type')"
+          required />
+        <dp-multiselect
+          v-model="currentProcedureType"
+          class="layout__item u-1-of-1 u-pl-0 u-mb display--inline-block"
+          label="name"
+          :options="procedureTypes"
+          required
+          track-by="id">
+          <template v-slot:option="props">
+            {{ props.option.name }}<br>
+            <span class="font-size-small">{{ props.option.description }}</span>
+          </template>
+        </dp-multiselect>
+        <input
+          type="hidden"
+          name="r_procedure_type"
+          :value="currentProcedureTypeId">
+      </div>
+<!--      {# There should always be at least one procedureType defined #}-->
```

Comment:
see above

Reply:
Done in [7c7706b](https://github.com/demos-europe/demosplan-core/pull/752/commits/7c7706b9bf34fcb618e07312e18830b6344b3921)